tasks/config.yml

```diff
@@ -17,3 +17,22 @@
     group: "{{ confluence_group }}"
     mode: '0644'
   notify: Confluence | Service restart
+
+- name: Confluence | Setting Catalina config
+  template:
+    src: templates/setenv.sh.j2
+    dest: "{{ confluence_home_path }}/{{ confluence_pkg }}/bin/setenv.sh"
+    owner: "{{ confluence_user }}"
+    group: "{{ confluence_group }}"
+    mode: '0644'
+  notify: Confluence | Service restart
+
+- name: Confluence | Setting logs location
```

Comment:
I would prefer to provide that config as a config template or file instead of modifying lines using regexps

Reply:
redone